src/Client/app/(admin)/invite-candidates/page.tsx

```
@@ -330,21 +333,14 @@ export default function Component() {
+        </div>
         <div className="p-2 m-2 flex justify-between items-center">
-          <Pagination
-            isCompact
-            showControls
-            showShadow
-            color="primary"
-            page={page}
-            total={totalPages}
-            onChange={setPage}
-          />
+         <div></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)